### ISSUE TITLE
The Compact Combobox name=one is missing a visible label.

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4689]
[Test URL]
https://5ec782484499f3fd69725086--vmware-clarity.netlify.app/combobox

[WCAG 2.1 Guideline]
Success Criterion 3.3.2 Labels or Instructions (Level A)
https://www.w3.org/TR/WCAG21/#labels-or-instructions
https://www.w3.org/WAI/WCAG21/Techniques/html/H44.html (This technique requires the labels to be visible)

[WCAG Conformance Level]
A

[OS]
Any

[User Agents/Assistive Technologies]
Any

[Component]
Combobox

[Description]
The Compact Combobox name=&quot;one&quot; is missing a visible label.

See the relevant HTML below:
&lt;clr-combobox name=&quot;one&quot; required=&quot;&quot; aria-label=&quot;one&quot; class=&quot;foo aria-required clr-combobox ng-pristine ng-invalid ng-touched&quot; id=&quot;clr-form-control-18-combobox&quot;&gt;&lt;div clrpopoveranchor=&quot;&quot; class=&quot;clr-combobox-wrapper clr-anchor&quot;&gt;&lt;!----&gt;&lt;span role=&quot;combobox&quot; aria-haspopup=&quot;listbox&quot; class=&quot;clr-combobox-input-wrapper&quot; aria-expanded=&quot;false&quot; aria-owns=&quot;clr-options-10&quot;&gt;&lt;input type=&quot;text&quot; autocomplete=&quot;off&quot; aria-autocomplete=&quot;list&quot; aria-multiline=&quot;false&quot; class=&quot;clr-input clr-combobox-input ng-pristine ng-valid ng-touched&quot; id=&quot;clr-form-control-18&quot; aria-controls=&quot;clr-options-10&quot;&gt;&lt;/span&gt;&lt;button clrpopoveropenclosebutton=&quot;&quot; type=&quot;button&quot; tabindex=&quot;-1&quot; class=&quot;clr-combobox-trigger clr-smart-open-close&quot;&gt;&lt;clr-icon shape=&quot;caret down&quot; size=&quot;12&quot; role=&quot;none&quot; style=&quot;width: 12px; height: 12px;&quot;&gt;&lt;svg version=&quot;1.1&quot; viewBox=&quot;0 0 36 36&quot; preserveAspectRatio=&quot;xMidYMid meet&quot; xmlns=&quot;http://www.w3.org/2000/svg&quot; xmlns:xlink=&quot;http://www.w3.org/1999/xlink&quot; focusable=&quot;false&quot; role=&quot;img&quot;&gt;&lt;path class=&quot;clr-i-outline clr-i-outline-path-1&quot; d=&quot;M29.52,22.52,18,10.6,6.48,22.52a1.7,1.7,0,0,0,2.45,2.36L18,15.49l9.08,9.39a1.7,1.7,0,0,0,2.45-2.36Z&quot;&gt;&lt;/path&gt;&lt;/svg&gt;&lt;/clr-icon&gt;&lt;/button&gt;&lt;/div&gt;&lt;!----&gt;&lt;/clr-combobox&gt;

[User Impact]
Users may not be able to visibly determine the accessible name of this combobox. 

[Severity (1 -10)]
10

[Steps to Reproduce]
- Go to the Test URL
- Observe that the Compact Combobox name=&quot;one&quot; does not have a visible label.

[Recommendation]
- Use a visible label for all form fields

**Comment by gnomeontherun** [comment]
This is just our demo app, closing as this is not a documentation case nor recommended, but we need to test that it still works in isolated mode.
